docroot/themes/custom/uids_base/scss/components/card.scss

```diff
@@ -306,6 +306,12 @@
   margin-top: .3rem;
 }
 
+.card__meta {
```

Comment:
We could probably move this back to UIDS4 with something like:

.card__meta {
  margin-top:.3rem;
  > * {
  margin-top: .3rem;
  }
}

I'll make a note in the UIDS4 issue.